### PR TITLE
relay: Fix createRequestHandler flow types

### PR DESCRIPTION
**Pull request by tbergq**
- Fix createEnvironment flow types

### CODE REVIEW

File: src/relay/src/createRequestHandler.js [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
 import type { RequestNode, Uploadables } from './types.flow';
 
-export type CacheConfig = {|
-  +force?: ?boolean,
-|};
-
 type GraphQLResponse = {|
-  +data?: null | { [key: string]: any, ... },
-  +errors?: $ReadOnlyArray<{ [key: string]: any, ... }>,
+  +data: PayloadData,
```
Comment (mrtnzlml) [inline src/relay/src/createRequestHandler.js:9]: Property `data` doesn't have to be in response. Is this type correct? 🤔

Comment (tbergq) [reply src/relay/src/createRequestHandler.js:9]: This is the type from `node_modules/relay-runtime/network/RelayNetworkTypes.js.flow`
```
export type GraphQLResponseWithData = {|
  +data: PayloadData,
  +errors?: Array<PayloadError>,
  +extensions?: PayloadExtensions,
  +label?: string,
  +path?: Array<string | number>,
|};
```

Comment (mrtnzlml) [reply src/relay/src/createRequestHandler.js:9]: I can see it's a bit more complicated:

```js
export type GraphQLResponse =
  | GraphQLSingularResponse
  | $ReadOnlyArray<GraphQLSingularResponse>;
```

https://github.com/facebook/relay/blob/26a4fc646cf9a2614ae95468efc90a619bb6c57b/packages/relay-runtime/network/RelayNetworkTypes.js#L44-L84

We already have this type somewhere actually. Maybe we could reuse it?

Comment (tbergq) [reply src/relay/src/createRequestHandler.js:9]: Yeah, that seems to work 👍 I will use it

Comment (tbergq) [reply src/relay/src/createRequestHandler.js]: 🚀

**Pull request merged**